/*
 * controller_core.h
 *
 *  Created on: May 5, 2012
 *      Author: Michael Anderson
 */

#ifndef CONTROLLER_CORE_H_
#define CONTROLLER_CORE_H_

#include <ac_leg_angle_sine_wave/controller_common.h>
#include <atrias_control/core_library.h>
#include <drl_library/drl_math.h>
#include <math.h>

float accumulator_length;
float accumulator_angle;

#endif /* CONTROLLER_CORE_H_ */
